core/new-gui/src/app/workspace/component/visualization-panel-content/visualization-panel-content.component.html

```diff
@@ -1,11 +1,9 @@
 <meta charset="utf-8">
 
-<div>
   <!-- this id should be identical to VisualizationPanelContentComponent.CHART_ID -->
   <div id="texera-result-chart-content">
   </div>
   <!-- this id should be identical to VisualizationPanelContentComponent.MAP_CONTAINER -->
-  <div id="texera-result-map-container">
+  <div *ngIf="displayMap" id="texera-result-map-container">
   </div>
-
-</div>
+  <iframe *ngIf="displayHTML" id="texera-result-html-content-container" [srcdoc]="htmlData"></iframe>
```

Comment:
Just adding a comment to confirm that you didn't remove the <div>, </div> at lines 3 and 11 by mistake.

Reply:
they are removed as I think the visual result without a parent container div looks better with all visualization types and charts